Sources/SwiftFrameCore/Workers/ImageWriter.swift

```diff
@@ -42,12 +43,12 @@ public final class ImageWriter {
         workGroup.wait()
     }
 
-    static func sliceImage(_ image: CGImage, with size: CGSize) throws -> [CGImage] {
+    static func sliceImage(_ image: CGImage, with size: CGSize, gapWidth: Int? = nil) throws -> [CGImage] {
```

Comment:
I think `ConfigProcessor` passes `gapWidth` down as a non-optional, right? Maybe we could keep it non-optional here as well to simplify the code?